doc/exts/pylint_messages.py

```diff
@@ -179,10 +186,17 @@ def _write_message_page(messages_dict: MessagesDict) -> None:
 {message.bad_code}
 {message.details}
 {message.related_links}
-
-Created by ``{message.checker}`` checker
 """
                 )
+                if message.checker_module.startswith("pylint.extensions."):
+                    stream.write(
+                        f"""
+.. note::
+  This check requires ``{message.checker_module}`` plugin to be loaded. See: :ref:`{message.checker_module}`
+
+"""
+                    )
+                stream.write(f"""Created by ``{message.checker}`` checker""")
```

Comment:
While we're at it. Originally I was thinking of adding a link to the code on Github here. Something like `https://github.com/PyCQA/pylint/blob/main/pylint/checkers/CHECKER_NAME` should work? The only thing we should be careful about is checkers in subdirectories.

Is this something you would want to explore in this PR as well?

Reply:
I will check that.

Reply:
Implemented